Makefile

```diff
@@ -49,7 +49,7 @@ check_pydocstyle:
 	pydocstyle $(python-packages)
 
 check_doc:
-	tools/check_doc.py
+	tools/check_doc.py doc/language_reference/index.rst doc/development_guide/index.rst doc/user_guide/index.rst
```

Comment:
Is there a reason to not check all documents by default? With an explicit list we may forget checking the doc if we add something.

Reply:
Changed, as discussed above.